.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  extends: [
+    "plugin:@typescript-eslint/recommended"
+  ],
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: "module"
+  },
+  root: true,
+  rules: {
+    "@typescript-eslint/array-type": [
```

Comment:
FWIW, I don't have strong feelings on this rule in particular, it was just part of the https://medium.com/@KevinBGreene/migrating-from-tslint-to-eslint-6382f8bd3b6 boilerplate.